apps/agreement/test/agreement/agreement_new_action.js

```diff
@@ -39,90 +43,156 @@ contract('Agreement', ([_, owner, submitter, someone]) => {
               })
 
               context('when the signer has enough balance', () => {
-                context('when the agreement settings did not change', () => {
-                  it('creates a new scheduled action', async () => {
-                    const currentSettingId = await disputable.getCurrentSettingId()
-                    const currentCollateralId = await disputable.getCurrentCollateralRequirementId()
-                    const { actionId, disputableActionId } = await disputable.newAction({ submitter, actionContext, stake, sign })
-
-                    const actionData = await disputable.getAction(actionId)
-                    assert.equal(actionData.disputable, disputable.disputable.address, 'disputable does not match')
-                    assert.equal(actionData.submitter, submitter, 'submitter does not match')
-                    assert.equal(actionData.context, actionContext, 'action context does not match')
-                    assert.isFalse(actionData.closed, 'action state does not match')
-                    assertBn(actionData.settingId, currentSettingId, 'setting ID does not match')
-                    assertBn(actionData.disputableActionId, disputableActionId, 'disputable action ID does not match')
-                    assertBn(actionData.collateralRequirementId, currentCollateralId, 'action collateral ID does not match')
+                const newActionFlow = (collateralTokenTransactionFeeAmount) => {
```

Comment:
Can we rename this to `transactionFeeAmount` simply?

Reply:
Ok. I named this way because the param only takes a non-zero value when the transaction fees token is staking collateral token.